libraries/conf_helpers.rb

```diff
@@ -46,7 +46,8 @@ def self.parse_config(conf_body)
         case ln
         when /^\s*\[([^#]+)\]\s*/
           current_section = Regexp.last_match[1]
-        when /^\s*([^#]+?)\s*=\s*([^#]+?)\s*(?:#.*)?$/
+          current_config[current_section] ||= {}
+        when /^\s*([^#]+?)\s*=\s*([^#\v]*?)\s*(?:#.*)?$/
```

Comment:
I'm curious what lead to this change in the second capturing group, to add a negative match on vertical tab?

(We probably should add line continuation parsing as well... I wonder if we need an actual parser generator...)...